_data/tools.yml

```
@@ -100,3 +100,25 @@ development:
       - name: Selenium
         documentation: https://www.seleniumhq.org/
     
+- focus: ESDC Standard Security Tools by Security Testing Type
```

We'll need to fix the tabbing so it loads properly, and the `by Security Testing Type` won't be needed as the `scope` will be appended to the title.
```suggestion
  - focus: ESDC Standard Security Tools
```